### PR TITLE
🧹  edited example bundle comments, changed one check

### PR DESCRIPTION
**Pull request by misterpantz**
Fixed typos
Made comments more readable

### CODE REVIEW

File: examples/example.mql.yaml [hunk 1]
```diff
@@ -2,30 +2,29 @@
...
   - uid: example1
     name: Example policy 1
     version: "1.0.0"
-    # If your policies has queries that have a "severity", you should set
-    # the scoring system to "highest impact" (like in this case). It makes
-    # sure that the lowest score (i.e. highest impact failure) is used as
-    # the overall score for the report.
+    # If your policy has queries with severities specify the
+    # "highest impact" scoring system. This system uses the lowest
+    # score (highest impact failure) as the overall report score.
```
Comment (tas50) [inline examples/example.mql.yaml]: We could really use a simple example here

Comment (misterpantz) [reply examples/example.mql.yaml]: Can you recommend one?

Comment (tas50) [reply examples/example.mql.yaml]: A smaller version of this scenario:

- You have a policy with 3 with failing queries:
  - 1 with impact of 10
  - 1 with impact of 30
  - 1 with impact of 90 (pretty critical)
- Your policy scores a 10 (100-90) and thus is an F

Comment (misterpantz) [reply examples/example.mql.yaml]: Thanks, that helps me. I changed the text based on this... but I didn't provide an example. I think that's a lot of info for a sample file. I'll make sure to describe it in the docs.

**Review by arlimus — APPROVED** [review]
Awesome improvements, thank you!! these examples are more :star2:

**Pull request merged**